.gitignore

```diff
@@ -37,3 +37,8 @@ public/css/semantic
 
 # .env
 .env
+*.psd
+*.png
```

Comment:
.png, assets/ and jquery are all fine to commit

Reply:
no.